applications/dashboard/models/class.importmodel.php

```diff
@@ -460,7 +460,7 @@ public function deleteFiles() {
     public function deleteOverwriteTables() {
         $tables = ['Activity', 'Category', 'Comment', 'Conversation', 'ConversationMessage',
             'Discussion', 'Draft', 'Invitation', 'Media', 'Message', 'Photo', 'Permission', 'Rank', 'Poll', 'PollOption', 'PollVote', 'Role', 'UserAuthentication',
-            'UserComment', 'UserConversation', 'UserDiscussion', 'UserMeta', 'UserRole'];
+            'UserComment', 'UserConversation', 'UserDiscussion', 'UserMeta', 'UserRole','Tag'];
```

Comment:
```suggestion
            'UserComment', 'UserConversation', 'UserDiscussion', 'UserMeta', 'UserRole', 'Tag'];
```

This is probably going to fail the linter. I'd recommend configuring your PHPStorm to use our coding standard https://docs.vanillaforums.com/developer/contributing/coding-standard-php/#codesniffer-phpstorm-integration

Reply:
You are right Adam. Thank you to point this out. I actually have code sniffer in my PHP storm, but I forget to use it.